### ISSUE TITLE
Update UI tests for new toolbar

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#22097]
The new toolbar changes a lot of a11y ids as well as the user experience. Before we enable the new toolbar by default all the UI tests should be updated as otherwise tests will fail.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10086)

**Comment by data-sync-user** [comment]
➤ Norberto Andres Furlan commented:

Is this a blocker for release?

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

Norberto Andres Furlan no, it’s not.

**Comment by data-sync-user** [comment]
➤ Chris Peterson commented:

Winnie says we can close this issue because the automation team has the updated UI tests ready. They will switch the UI tests when the new toolbar goes live.
